### PR TITLE
Add search query parameters to GLOBALS object

### PR DESCRIPTION
**Pull request by MelisaIm**
Added the features for Day 1 requirements

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,16 +9,21 @@
 		<form action="whack-a-mole.html" method="get">
 			<fieldset>
 				<legend>Mole Properties</legend>
-				<input type="number" value="3" name="MOLE_UP_MIN"/>
+				<label>Min Moles</label>
+					<input type="number" value="3" name="MOLE_UP_MIN"/>
```
Comment (warandoates) [inline index.html]: Why is this indented?

File: index.html [hunk 2]
```diff
@@ -9,16 +9,21 @@
...
+					<label>Max Moles</label>
 				<input type="number" value="5" name="MOLE_UP_MAX"/>
 			</fieldset>
 			<fieldset>
 				<legend>Game Modifiers</legend>
+				<label>Number Of Rounds</label>
 				<input type="number" value="10" name="NUMBER_OF_ROUNDS"/>
```
Comment (warandoates) [inline index.html]: please add id for each input

File: index.html [hunk 3]
```diff
@@ -9,16 +9,21 @@
...
 				<legend>Mole Properties</legend>
-				<input type="number" value="3" name="MOLE_UP_MIN"/>
+				<label>Min Moles</label>
+					<input type="number" value="3" name="MOLE_UP_MIN"/>
+					<label>Max Moles</label>
 				<input type="number" value="5" name="MOLE_UP_MAX"/>
 			</fieldset>
 			<fieldset>
 				<legend>Game Modifiers</legend>
+				<label>Number Of Rounds</label>
```
Comment (warandoates) [inline index.html]: set the for attribute for each input

File: javascript/main.js [hunk 4]
```diff
@@ -22,39 +22,38 @@ window.onload = function(){
 //   MOLES_PER_ROUND
 //   ROUND_COOLDOWN (in seconds)
 function readQueryParams() {
-    /* YOUR CODE HERE */
+	let searchArr = document.location.search.replace(/\?/, '').split('&');
+	searchArr.forEach(query => {
+		query = query.split('=');
+		GLOBALS[query[0]] = parseInt(query[1]);
+	});
+	console.log(GLOBALS);
```
Comment (warandoates) [inline javascript/main.js]: please remove

**Review by warandoates — CHANGES_REQUESTED** [review]
good job so far

**Comment by MelisaIm** [comment]
Hey Michael, I fixed the bugs you requested.

**Review by warandoates — APPROVED** [review]

**Pull request merged**